### DISCUSSION TITLE
06/11 피어세션

### DISCUSSION THREAD

**Discussion by joochanhyeong** [bcaitech1/p4-dkt-DKTJHGSD#103] [Meeting Notes]
< TMI >

[ 이태환 ] : 공부 열심히 하고 있는데, 어머니가 사과를 가져다주셔서 맛있게 먹었 당.

[ 이수연 ] : 면접질문 채널에 공유했ㄷ ㅏ. 면접 썰 대개방. 문제마다 내가 답변어떻게 했는지 말해줌. 근데 기업이랑 느낌이 많이 달랐(좀 깊게 꼬리질문!!). 그래도 재밌었당. 

[ 이준호 ] : 주말에 폴로셔츠 하나 살거야. 살쪄가지고 평상복 입기힘들어. 범계가서 살거야. 오랜만에 옷을 사서 기분이 좋아. 코로나때문에 사람도 안만나서 그런지 옷 별로 안사게 되는 듯. 근데 그만큼 옷을 안사서 돈이 있을 줄 알았는데 대부분 배달음식으로 갔더라. 

[ 백서윤 ] : 어제 그거 뿌리고 잤는데 냄새 좋아서 기분은 좋은데 잠은 안왔어. 일 생각했어. bias 나쁜놈. 옷은 안사게 됐는데 잠옷을 많이 사게됐다. 원래 한 두개 였는데(입고싶은ㄹ 날 입고) 잠옷 쇼핑에 빠졌다. 

[ 주찬형 ] : 오늘은 카페 왔는데 조금 특이하게 교회에 있는 카페다. 별로 안와봤는데 사장님이 기독교 신자신지 카페옆에 기도실이 있어서 조금 엄숙하다. 그래서 소리를 못내겠어요... 아직 저는 수원이예요.. 옷 몇개 사가고 같이 동거하는 친구랑 정도 많이 들어서 같이 밤새고 야식먹고 하니깐 재밌더구라구요. 저도 편의점에서 팬티를 샀습니다. 요즘 야식을 많이 먹어서 살이 찐게 아니라 일어나면 몸이 찌뿌둥 하다. 집이 아침이 되면 새들이 둥지를 트는지 새들이 엄청 짹짹대서 그거 때문에 깬다. 아주 열불난다잉!!!!!

[ 진소정 ] : 머리를 잘 못말려서 여름에 에어컨 켜서 머리가 차가월. 어제 찬형이가 자기 한거 물어봐서 논문 조사하다가 구현 잘못된거 발견.. 근데 고치기 전이 더 성능이 좋아서 답답했따!!!!!!!!

< review >

 

**잘했던것, 좋았던것, 계속할것**

- kfold ensemble
- pretraining 시도
- last query
- SAINT
- testid 별로 나눠서 학습
- sum feature 와 mean feature를 같이 쓴 것
- train + test 데이터 합해서 학습 및 통계량 측정

**잘못했던것, 아쉬운것, 부족한것 , 개선방향**

- 해당 논문 구현 뒤 논문에서 제시한 하이퍼 파라미터로 학습을 시도 했으나 결과가 좋지 않았다..
- str(x)의 문제를 3주차에 발견한 것...

**도전할것, 시도할것**

- LGBM으로 성능 올리기
- LANA model로 성능 올리기
- pseudo labeling
- T-Fixup 적용시켜 학습시키기
- seed ensemble
- OOF stacking ensemble

**키워드(공부한것, 알게된것, 느낀점)**

- feature engineering
- DKT task에 적합한 sequential data processing, 모델들
- AUC랑 accuracy 차이!!
- 대용량 데이터 다루기

< 제출 review >

- 미리미리 단일 모델 kfold 돌려놓장

< progress >

- activation idea —> MLP 나 dropout, layer normalization 사이에 activation으로 leak-relu 사용해보기
- LGBM 정상 작동을 위해 feature 수작업 중 - 이태환
- 대회 마감 고려해서 성능 개선 전략 위주 역할 분담 진행 —> stacking ensemble, t-fixup, pseudo labeling 역할 분담 완료
- LANA, SAINT, SAKT, convBERT, lstmattn 위주로 riiid pretrain 미리 준비, 모델 별 각 학습 결과는 추후 공유 예정. LANA랑 SAKT는 학습 중, SAINT는 파라미터 탐색을 어느정도 정상화 시켜놓고 학습 예정.

< 네트워킹 데이 준비 >

- 공유 readme 작성 —> HackMd 협업 툴로 작성해보기!
